client/gutenberg/extensions/publicize/panel.jsx

```diff
@@ -33,27 +32,17 @@ import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
 
 const PublicizePanel = ( { connections, refreshConnections } ) => (
 	<PostTypeSupportCheck supportKeys="publicize">
```

Comment:
Shouldn't we also move this wrapper to `./editor.js`? IIUC, if the current post type doesn't support `publicize`, this PR would still add an (empty) publicize item to both the Jetpack toolbar, and the pre-publish panel.

Reply:
(I just confirmed this hypothesis with a `feedback` post type.)

Reply:
Good call 👍 Done in 040b131a29337cdbef96ee34d143b1a1b6b06a9d.